app/states/unified_database.py

```
@@ -0,0 +1,1208 @@
+        Get all officers for a committee, grouped by role.
+        """
+        committee_persons = self.get_committee_persons(committee_id, active_only=active_only)
+        officers = {}
+        for cp in committee_persons:
+            if cp.role not in officers:
+                officers[cp.role] = []
+            officers[cp.role].append(cp)
+        return officers
+
+    def link_transaction_to_committee_role(self, transaction_person_id: int, committee_person_id: int, 
+                                         user: Optional[str] = None, notes: Optional[str] = None) -> bool:
```

Ruff (0.14.7)

Unused method argument: `user`

(ARG002)